app/renderer/css/preference.css

```diff
@@ -605,6 +605,8 @@ input.toggle-round:checked+label:after {
 }
 
 .tip {
+    display: flex;
```

Comment:
We don't want it to be centred, please remove these two lines.

Reply:
@akashnimare  removed the css, you can review it